Libraries/LibJS/Interpreter.h

```diff
@@ -59,6 +59,7 @@ struct ScopeFrame {
 struct CallFrame {
     Value this_value;
     Vector<Value> arguments;
+    bool is_constructor;
```

Comment:
Please always initialize primitive types explicitly:
```suggestion
    bool is_constructor { false };
```